tests/test_fluxModelSet.py

```diff
@@ -0,0 +1,37 @@
+import lsst.utils
+import lsst.utils.tests
+from pfs.datamodel.pfsSimpleSpectrum import PfsSimpleSpectrum
+from pfs.drp.stella.fluxModelSet import FluxModelSet
+from pfs.drp.stella.tests import runTests
+
+import os
+import unittest
+
+try:
+    dataDir = lsst.utils.getPackageDir("fluxmodeldata")
+except LookupError:
+    dataDir = None
+
+
+@unittest.skipIf(dataDir is None, "fluxmodeldata not setup")
+class FluxModelSetTestCase(lsst.utils.tests.TestCase):
+    def setUp(self):
+        self.fluxModelSet = FluxModelSet(os.path.join(dataDir))
+
+    def testGetSpectrum(self):
+        """Test getFileName()
+        """
+        parameters = self.fluxModelSet.parameters
```

Comment:
Check that this contains the expected list of parameters?

Reply:
If any of the expected parameters ("teff", "logg", "m", "alpha") do not exist, an exception will be raised from below, which makes the test fail. I think this is already sufficient.